test/e2e/shared/suite.go

```diff
@@ -37,9 +37,9 @@ import (
 	. "github.com/onsi/gomega"
 	"sigs.k8s.io/yaml"
 
+	"sigs.k8s.io/cluster-api-provider-aws/v2/test/helpers/kubernetesversions"
```

Comment:
We could add a comment here saying this is a temporary workaround until a new CAPI v1.8.5 with https://github.com/kubernetes-sigs/cluster-api/pull/11307 included